var/spack/repos/builtin/packages/mpt/package.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+
+class mpt(Package):
+    """HPE MPI is HPE's implementation of 
+    the Message Passing Interface (MPI) standard.
+ 
+    Note: HPE MPI is proprietry software. Spack will search your
+    current directory for the download file. Alternatively, add this file to a
+    mirror so that Spack can find it. For instructions on how to set up a
+    mirror, see http://spack.readthedocs.io/en/latest/mirrors.html"""
+
+    homepage = ""
```

Comment:
Looks like there are a few homepages available to choose from.

Reply:
Is `https://buy.hpe.com/us/en/software/high-performance-computing-software/hpe-message-passing-interface-mpi/p/1010144155` appropriate ?

Reply:
Sure!